tests/cli_test.py

```diff
@@ -761,12 +761,25 @@ def test_configure_command_create_vdir(runner):
     assert '{}/khal/calendars/private1' .format(runner.xdg_data_home) in actual_config
 
 
+def cleanup(paths):
+    """reset permissions of all files and folders in `paths` to 666 resp. 777"""
+    for path in paths:
+        if os.path.exists(path):
+            os.chmod(str(path), 0o777)
+        for dirpath, dirnames, filenames in os.walk(path):
+            os.chmod(str(dirpath), 0o777)
+            for filename in filenames:
+                os.chmod(str(os.path.join(dirpath, filename)), 0o666)
+
+
 def test_configure_command_cannot_write_config_file(runner):
     runner = runner()
     runner.config_file.remove()
     os.chmod(str(runner.xdg_config_home), 555)
     result = runner.invoke(main_khal, ['configure'], input=choices())
     assert result.exit_code == 1
+    # make sure pytest can clean up behind us
+    cleanup([runner.xdg_config_home, runner.xdg_data_home])
```

Comment:
Do we need to cleanup `xdg_data_home`? We didn't change it in this test.

Reply:
removed